### ISSUE TITLE
Failed to load camera using Convnet Classifier

### ISSUE THREAD

**Issue by DA4KCODE** [ml4a/ml4a-ofx#35]
Hi, I tried to use Convnet Classifier for my project. I followed the guidelines and successfully built the app using Xcode. But the camera seemed not work properly(as follow: all balck)
<img width="1400" alt="Screen Shot 2019-04-21 at 10 51 41 PM" src="https://user-images.githubusercontent.com/45328689/56475444-a0f12980-6488-11e9-8c23-168efb0f49fd.png">
I am not sure what it's wrong.
I duplicated the app and renamed it.
<img width="710" alt="Screen Shot 2019-04-21 at 10 52 16 PM" src="https://user-images.githubusercontent.com/45328689/56475472-f4fc0e00-6488-11e9-8887-9aeb8680652a.png">
and I added the image-net-2012 file inside the bin>data
<img width="1191" alt="Screen Shot 2019-04-21 at 10 52 32 PM" src="https://user-images.githubusercontent.com/45328689/56475497-4f956a00-6489-11e9-8b60-f9cb2f423fc7.png">
I also followed Read.me and added two addons inside openframeworks>addons
<img width="1074" alt="Screen Shot 2019-04-21 at 11 02 08 PM" src="https://user-images.githubusercontent.com/45328689/56475519-9d11d700-6489-11e9-8414-aaa20608e2ce.png">

Everything seems correct but it just didn’t work out. It will be really helpful if you can point out some mistakes! Thank you so much!!!

**Comment by genekogan** [comment]
can you get any openframeworks examples that use the camera to work? if not, the problem is your camera, not ml4a-ofx. the addons do not affect the functioning of the camera.

**Comment by DA4KCODE** [comment]
> can you get any openframeworks examples that use the camera to work? if not, the problem is your camera, not ml4a-ofx. the addons do not affect the functioning of the camera.

Hi, thank you so much for the reply. I tried convnet OSC and it didn't work as well. I also tried some example on Processing and it looks like this:
<img width="914" alt="Screen Shot 2019-04-22 at 11 30 32 AM" src="https://user-images.githubusercontent.com/45328689/56494352-3bd21e00-64f2-11e9-8f0e-bc1a202fe56a.png">
I installed Snap Camera few days ago and I think maybe it affect the functioning of the webcam. The webcam functions on Processing only when I open the Snap Camera at the same time. But the open frameworks app stiil don't work properly.

If you have any suggestions to fix this problem, it will be really helpful. Thx a lot:)
